content/about/notes/1.1/index.md

```diff
@@ -1,32 +1,101 @@
 ---
 title: Istio 1.1
-publishdate: 2019-03-01
+publishdate: 2019-03-12
 icon: notes
 ---
 
-TODO announcement
+We're proud to release Istio 1.1! It's been a while since 1.0 and we've done some significant improvements to make
+the overall product more reliable, more efficient, and more scalable.
+
+These release notes describe what's different between Istio 1.0 and Istio 1.1.
 
 {{< relnote_links >}}
 
-## Policies and Telemetry
+## Upgrades
+
+- **Helm Changes**. 
+TBD: need full content & links
+
+- **TBD: TITLE**, Control plane and data plane requested resource changes
+TBD: need full content & links
+
+## Traffic management
+
+- **Allow All Outbound Traffic**. Allow all outbound traffic by default now.   To enforce no outbound traffic at mesh wide, configure ```global.outboundTrafficPolicy.mode``` to ```REGISTRY_ONLY```
```

Comment:
Where would a reader configure ```global.outboundTrafficPolicy.mode``? What commands would the readers need to run and where? It would be better to link to the task detailing how to perform this configuration.

Reply:
agreed, which is https://preliminary.istio.io/docs/tasks/traffic-management/egress/#change-back-to-the-blocking-by-default-policy